numpy/lib/arraysetops.py

```diff
@@ -527,36 +527,36 @@ def isin(elements, test_elements, assume_unique=False, invert=False):
     Notes
     -----
 
-    `isin` can be considered as an element-wise function version of the
-    python keyword `in`. ``isin(a, b)`` is roughly equivalent to
+    `isin` is an element-wise function version of the python keyword `in`.
+    ``isin(a, b)`` is roughly equivalent to
     ``np.array([item in b for item in a])`` if `a` and `b` are 1-D sequences.
-    
-    If `test_elements` is a set (or other non-sequence collection) it will 
-    be converted to an object array with one element, rather than an array 
-    of the values contained in `test_elements`. Converting the set to 
+
+    If `test_elements` is a set (or other non-sequence collection) it will
+    be converted to an object array with one element, rather than an array
+    of the values contained in `test_elements`. Converting the set to
     a list usually gives the desired behavior.
 
     .. versionadded:: 1.13.0
 
     Examples
     --------
-    >>> elements = np.array([[0, 2], [4, 6]])
+    >>> element = np.array([[0, 2], [4, 6]])
     >>> test_elements = [1, 2, 4, 8]
-    >>> mask = np.isin(elements, test_elements)
+    >>> mask = np.isin(element, test_elements)
     >>> mask
     array([[ False,  True],
            [ True,  False]], dtype=bool)
-    >>> elements[mask]
+    >>> element[mask]
     array([2, 4])
-    >>> mask = np.isin(elements, test_elements, invert=True)
+    >>> mask = np.isin(element, test_elements, invert=True)
     >>> mask
     array([[ True, False],
            [ False, True]], dtype=bool)
-    >>> elements[mask]
+    >>> element[mask]
     array([0, 6])"""
-    elements = np.array(elements)
-    return in1d(elements, test_elements, assume_unique=assume_unique,
-                invert=invert).reshape(elements.shape)
+    element = np.array(element)
```

Comment:
Should be `asarray`, or `asanyarray` if you think this will worked for masked arrays